app/ui-message/client/message.js

```diff
@@ -385,11 +385,16 @@ Template.message.helpers({
 		// const subscription = Subscriptions.findOne({ rid: this.rid });
 		return subscription && subscription.name;
 	},
+	isEphemeral() {
+		const { room } = this;
+		return room.t === 'e';
```

Comment:
Remember to check if room exists before using it :D (or use `room?.t` to be safe)